src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -7,14 +7,61 @@ import org.ergoplatform.nodeView.state.{ErgoState, UtxoState}
 import org.ergoplatform.settings.ErgoSettings
 import scorex.core.transaction.MemoryPool
 import scorex.core.transaction.state.TransactionValidation
-import scorex.util.ModifierId
+import scorex.util.{ModifierId, bytesToId}
 
 import scala.util.Try
 
+object MemPoolStatisticsParams {
+  val nHistogramBins = 60  /* one hour */
+  val measurementIntervalMsec = 60 * 1000 /* one hour */
+}
+
+case class FeeHistogramBin(var nTxns: Int, var totalFee: Long)
+
+case class MemPoolStatistics() {
```

Comment:
Case classes without args is anti-pattern. Why not use `case object` then?
Also please add ScalaDoc.